internal/artifacts/module_arch.go

```diff
@@ -120,7 +120,10 @@ func ExecuteSoloBuild(source, target string, extensions []string, modulesNames [
 	return nil
 }
 
-func checkBuildResultsConflicts(mtaObj *mta.MTA, source, target string, extensions []string, modulesNames []string, wdGetter func() (string, error)) error {
+// Fail-fast check on modules whose build results are resolved (not glob patterns),
```

Comment:
This comment should be in the call to the function, not the function definition

Reply:
moved